### PR TITLE
Fixing LHCb init code

### PR DESCRIPTION
**Pull request by rob-c**
The latest LHCb code is not safe for me and bails out in a bad way.

I've updated the code and I think this needs integrating into 6.2.1.

### CODE REVIEW

**Comment by rob-c** [comment]
I still need to update the lhcb-prepare script. I'll finish this now

**Comment by rob-c** [comment]
@mesmith75 Any objections to merging this? which branch is best?

**Comment by mesmith75** [comment]
It looks ok. It could probably go into the release/6.2.1 branch. Then I can rerun the release tests.

File: python/GangaLHCb/__init__.py [hunk 1]
```diff
@@ -68,19 +68,22 @@ def _store_dirac_environment():
...
-        logger.info("Storing new LHCbDirac environment (%s:%s)" % (str(diracversion), str(platform)))
-      except (OSError, IOError, TypeError):
-        logger.error("Unable to store LHCbDirac environment")
-        raise
+        cmd =  'lb-run LHCBDIRAC {version} python -c "import os; print(dict(os.environ))"'.format(version=diracversion)
+        env = execute(cmd)
+        if isinstance(env, str):
+            try:
+               env_temp = eval(env)
+               env = env_temp
```
Comment (milliams) [inline python/GangaLHCb/__init__.py]: Why was this split on to two separate lines?

Comment (rob-c) [inline python/GangaLHCb/__init__.py]: So when debugging or returning what the env was its quickly clear what the value of env should be.

**Pull request merged**